npm-debug.log

```diff
@@ -0,0 +1,45 @@
+0 info it worked if it ends with ok
```

Comment:
I dunno why I was reading this commit but this caught my attention :): `npm-debug.log`